### PR TITLE
CNV-32699: Customize IT layouts

### PR DESCRIPTION
**Pull request by avivtur**
## 📝 Description

Adding layouts to for the Customize VM from IT flow

## 🎥 Demo

After:

https://github.com/kubevirt-ui/kubevirt-plugin/assets/67270715/76d5eebd-0a39-4334-b654-a2b6e713a3fc

### CODE REVIEW

File: src/views/catalog/CustomizeInstanceTypeVirtualMachine/tabs/yaml/YamlTab.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import { Alert, AlertVariant, Bullseye } from '@patternfly/react-core';
+
+import './YamlTab.scss';
+
+const YamlTab: FC = () => {
+  const { t } = useKubevirtTranslation();
+  const { setVM, vm } = useInstanceTypeVMStore();
+  const [error, setError] = useState<Error>(null);
+
+  const onSave = (yaml: string) => {
```
Comment (metalice) [inline src/views/catalog/CustomizeInstanceTypeVirtualMachine/tabs/yaml/YamlTab.tsx:19]: can be async

File: src/views/catalog/CustomizeInstanceTypeVirtualMachine/tabs/yaml/YamlTab.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    setVM(load(yaml) as V1VirtualMachine)
+      .then(() => {
+        setError(null);
+      })
+      .catch((apiError) => {
+        setError(apiError);
+      });
+  };
+
+  const loading = (
```
Comment (metalice) [inline src/views/catalog/CustomizeInstanceTypeVirtualMachine/tabs/yaml/YamlTab.tsx:30]: redundent

File: src/views/catalog/CustomizeInstanceTypeVirtualMachine/components/CustomizeITVMFooter.tsx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+                  })
+                    .then((createdVM) => {
+                      if (secretOption === SecretSelectionOption.addNew) {
+                        createSSHSecret(sshPubKey, sshSecretName, vmNamespaceTarget);
+                      }
+                      createHeadlessService(createdVM);
+                      navigate(getResourceUrl({ model: VirtualMachineModel, resource: vm }));
+                    })
+                    .catch(setError)
+                    .finally(() => setIsSubmitting(false));
```
Comment (metalice) [inline src/views/catalog/CustomizeInstanceTypeVirtualMachine/components/CustomizeITVMFooter.tsx:81]: no need for return and can be async

File: src/views/catalog/CustomizeInstanceTypeVirtualMachine/CustomizeInstanceTypeVirtualMachine.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+import CustomizeITVMFooter from './components/CustomizeITVMFooter';
+import CustomizeITVMHeader from './components/CustomizeITVMHeader';
+import { pages } from './utils/constants';
+
+const CustomizeInstanceTypeVirtualMachine: FC = () => {
+  const { vm } = useInstanceTypeVMStore();
+
+  return (
+    <>
```
Comment (metalice) [inline src/views/catalog/CustomizeInstanceTypeVirtualMachine/CustomizeInstanceTypeVirtualMachine.tsx:15]: redundent

File: src/utils/components/ErrorAlert/ErrorAlert.tsx [hunk 5]
```diff
@@ -0,0 +1,32 @@
+import React, { FC } from 'react';
+import { Link } from 'react-router-dom-v5-compat';
+
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { isEmpty } from '@kubevirt-utils/utils/utils';
+import { Alert, AlertVariant, Stack, StackItem } from '@patternfly/react-core';
+
+type ErrorAlertProps = {
+  error: any | Error;
```
Comment (metalice) [inline src/utils/components/ErrorAlert/ErrorAlert.tsx:9]: the any is needed?

File: src/views/catalog/CreateFromInstanceTypes/components/CreateVMFooter/CreateVMFooter.tsx [hunk 6]
```diff
@@ -105,23 +105,33 @@ const CreateVMFooter: FC = () => {
       .finally(() => setIsSubmitting(false));
   };
 
+  const handleCustomize = () => {
```
Comment (metalice) [inline src/views/catalog/CreateFromInstanceTypes/components/CreateVMFooter/CreateVMFooter.tsx:108]: can be async

File: src/utils/components/ErrorAlert/ErrorAlert.tsx (hunk 5 above)
Comment (avivtur) [reply src/utils/components/ErrorAlert/ErrorAlert.tsx:9]: yes for the href attribute

**Pull request merged**